pages/blog/march-2021-at-asyncapi.md

```diff
@@ -0,0 +1,135 @@
+- [AsyncAPI joins Linux Foundation](http://localhost:8080/blog/asyncapi-joins-linux-foundation)
+- [Linux Foundation Will Host AsyncAPI to Support Growth and Collaboration for Industry's Fastest-Growing API Spec](https://www.linuxfoundation.org/en/press-release/linux-foundation-will-host-asyncapi-to-support-growth-and-collaboration-for-industrys-fastest-growing-api-spec/)
+- [The Linux Foundation Announces Hosting of AsyncAPI](https://www.infoq.com/news/2021/03/linux-foundation-hosts-asyncapi/)
+- [AsyncAPI Looks to Unify API Workflow under Linux Foundation](https://thenewstack.io/asyncapi-looks-to-unify-api-workflow-under-linux-foundation/)
+
+### FAQ
+
+Let me provide more context in FAQ style:
+
+#### Is Linux Foundation taking control over AsyncAPI
+
+No. AsyncAPI Initiative runs under [open governance model](https://github.com/asyncapi/.github/blob/master/CHARTER.md) and is community-driven. LF assures the project's intellectual property (IP), and related assets do not belong to any company or individual. [Fran Mendez](https://twitter.com/fmvilas) does not retire. He only handed over rights to the project to the foundation to assure the community that it is completely safe for all to use the spec and its tooling.
+
+It also means our [GitHub organization](https://github.com/asyncapi) is not going anywhere. Nothing changes.
+
+#### Do we need to sign some CLA now to contribute
+
+We do not need a contributor license agreement (CLA) on repositories for our tools. We might need to set up [EasyCLA](https://easycla.lfx.linuxfoundation.org/#/) for the repository where we have the specification. We need to clarify it in the long run—nothing to worry about until we set it up. In the end, the only reason to set it up is to secure the community from a situation that some contributor (or his company) claims rights to some part of the specification they contributed to.
+
+#### How much money you need
+
+Glad you asked. We joined the foundation with an open governance model that favors active contributors over sponsors. As a result, we do not have a setup where we can assure significant income in exchange for voting rights. In other words, we do not have a financial founding associated with joining the foundation. We still need your financial support. Go to our [Open Collective profile](https://opencollective.com/asyncapi) and drop some coins. Let us know if I can help to peach about AsyncAPI in your company for some extra money.
+
+#### How can we help
+
+Except for regular contributions, we need help setting up tooling for our open governance model. We need to:
+- Automate a process of collecting information about TSC members
+- Put information about current TSC members on the AsyncAPI website
+- Figure out tooling for the website
+- Setup CODEOWNERS and VOTING files in all the repositories
+
+Everyone with a different set of skills is welcome. Just contact me.
```

Comment:
Should we add a link to Slack on the "Just contact me"? I'm sure many people still will not know how to reach out to us 😄

Reply:
done